src/ocl/ocl.client.ts

```diff
@@ -10,57 +10,108 @@ import { ProductNotFoundException } from '../common/exceptions/product-code-does
 import { OclClientException } from '../common/exceptions/ocl-client.exception';
```

Comment:
just a suggestion, instead of importing exceptions from different files, you can as well just create an index file in the exception folder. Then you can be referencing the index for all imports